app/authorities/sufia/authorities/local/find_works.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia::Authorities
+  class Local::FindWorks < Qa::Authorities::Base
+    attr_reader :q
```

Comment:
@andjsmit This is because you are attempting to https://github.com/projecthydra/sufia/pull/3015/files#diff-70cc36be61e57f96ec44887d4d62aa0fR120 register this as a subauthority. However it is not a subauthority. It is an authority in its own right.